externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/SparkSQLEngine.scala

```diff
@@ -83,6 +85,9 @@ object SparkSQLEngine extends Logging {
 
   private val countDownLatch = new CountDownLatch(1)
 
+  implicit private val ec: ExecutionContext =
+    ExecutionContext.fromExecutor(Executors.newFixedThreadPool(1))
```

Comment:
we need shutdown this executor, otherwire will cause thread leak

Reply:
And it doesn't make sense for all creations to share one thread

Reply:
In my opinion, for different engines, the createpark method will be executed in different processes and only once. 
Is there a problem with this understanding?

Reply:
You are right, the last comment is wrong, ignore it.